baker/SiteBaker.tsx

```diff
@@ -351,8 +351,21 @@ export class SiteBaker {
                 name: `✅ Prefetched ${Object.values(publishedExplorersBySlug).length} explorers`,
             })
 
+            // Get all grapher links from the database so that we only prefetch the ones that are actually in use
+            // 2024-06-25 before/after: 6266/2194
+            const grapherLinks = await db
+                .getGrapherLinkTargets(knex)
+                .then((rows) => rows.map((row) => row.target))
+                .then((targets) => new Set(targets))
+
             // Includes redirects
-            const publishedChartsRaw = await mapSlugsToConfigs(knex)
+            const publishedChartsRaw = await mapSlugsToConfigs(knex).then(
+                (configs) => {
+                    return configs.filter((config) =>
+                        grapherLinks.has(config.slug)
+                    )
+                }
+            )
```

Comment:
Arguably it would be nicer to write this as a single query but it's clearly an improvement over the status quo so totally fine to ship as is :)